digger/builds/__init__.py

```diff
@@ -22,8 +23,12 @@
     # then we can be sure that it is a Cordova project.
     # we also need to check if we want to build an IOS app or Android app for the Cordova.
     # for now, we don't do that and support Android only.
-    'is_valid': lambda path: os.path.exists('%s/config.xml' % path),
+    'is_valid': lambda path: os.path.exists('%s/www/config.xml' % path) or os.path.exists('%s/config.xml' % path),
```

Comment:
Cordova version 3.3.31 and bellow uses the config.xml file in www folder, other versions newer than that uses config.xml in root folder: [https://cordova.apache.org/docs/en/latest/config_ref/](https://cordova.apache.org/docs/en/latest/config_ref/)